go/service/chat_local.go

```diff
@@ -511,6 +518,13 @@ func (h *chatLocalHandler) localizeConversation(
 		return chat1.ConversationLocal{}, err
 	}
 
+	// TODO need this more validation to make sure it's not a trick?
+	conversationLocal.Info.Usernames = utils.ReorderParticipants(
```

Comment:
The server could send uids that are not in the tlf, but `ReorderParticipants` defends against that, so I think there's no problem now. Will delete this comment.